docs/capi-aws.md

```diff
@@ -67,79 +65,141 @@ metadata:
 
 ```shell
 % kubectl get cluster,machine
 NAME                                    PHASE         AGE   VERSION
-cluster.cluster.x-k8s.io/k0s-aws-test   Provisioned   46m   
+cluster.cluster.x-k8s.io/k0s-aws-test   Provisioned   16m   
+
+NAME                                         CLUSTER        NODENAME   PROVIDERID                                 PHASE         AGE   VERSION
+machine.cluster.x-k8s.io/k0s-aws-test-md-0   k0s-aws-test              aws:///eu-central-1a/i-05f2de7da41dc542a   Provisioned   16m   
+```
 
-NAME                                      CLUSTER        NODENAME   PROVIDERID                                 PHASE         AGE   VERSION
-machine.cluster.x-k8s.io/k0s-aws-test-0   k0s-aws-test              aws:///eu-central-1a/i-05f2de7da41dc542a   Provisioned   46m   
+You can also check the status of the cluster deployment with `clusterctl`:
+```shell
+% clusterctl describe cluster  
+NAME                                                   READY  SEVERITY  REASON                    SINCE  MESSAGE          
+Cluster/k0s-aws-test                                   True                                       25m                      
+├─ClusterInfrastructure - AWSCluster/k0s-aws-test                                                                             
+├─ControlPlane - K0smotronControlPlane/k0s-aws-test-cp                                                                           
+└─Workers                                                                                                                  
+  └─Other
 ```
 
+### Networking Options
+k0smotron, running in a management cluster in AWS, supports flexible networking options, allowing you to choose between Network Load Balancers (NLB) and Classic Elastic Load Balancers (ELB) based on your requirements for exposing the Control Planes.
+
+If you prefer using an NLB instead of ELB, you must specify annotations for the Service in the `k0smotronControlPlane`. These annotations guide the AWS Cloud Controller Manager (CCM) or the AWS Load Balancer Controller to create the respective services.
+
+```yaml
+ [...] 
+  service:
+    type: LoadBalancer
+    apiPort: 6443
+    konnectivityPort: 8132
+    annotations:
+      service.beta.kubernetes.io/aws-load-balancer-type: "nlb"
+```
+
+For scenarios involving Classic ELBs or NLBs without special options, the AWS CCM can be utilized.
+
+#### Internal NLB
+If you aim to use the NLB and set the schema to `internal`, the target group attribute `preserve_client_ip.enabled=false` is required due to "hairpinning" or "NAT loopback". In such cases, the AWS CCM cannot be used because it doesn't support setting Target Group Attributes. Therefore, the AWS Load Balancer Controller, which has the ability to set Target Group Attributes, becomes necessary.
```

Comment:
Add a link to the AWS LB controller setup instructions.

Reply:
Yes, makes sense.